scripts/tasks/verifyContracts.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  DEX_FACTORY,
+  DEX_ROUTER,
+  GAMMA_CONTROLLER,
+  GNOSIS_EASY_AUCTION,
+  LDO_ADDRESS,
+  MARGIN_POOL,
+  OTOKEN_FACTORY,
+  STETH_ETH_CRV_POOL,
+  USDC_ADDRESS,
+  WETH_ADDRESS,
+  WSTETH_ADDRESS,
+} from "../../constants/constants";
+
+const main = async (_taskArgs: any, { deployments, network, run }) => {
```

Comment:
We should be linting no explicted `any`.

I think you can type tasks args like:
```
import { TaskArguments } from "hardhat/types";

taskArgs: TaskArguments
```